src/TelemetryChannels/ServerTelemetryChannel/Shared/AdaptiveSamplingTelemetryProcessor.cs

```diff
@@ -71,6 +71,16 @@ public string ExcludedTypes
         }
 
         /// <summary>
+        /// Gets or sets a semicolon separated list of telemetry types that should be sampled. All types are sampled when left empty.
+        /// </summary>
+        public string IncludedTypes
+        {
+            get { return this.samplingProcessor.IncludedTypes; }
```

Comment:
The wording might be a bit off? 
ExcludedTypes take priority, so including an excluded type does nothing. But other than that they work separately and aren't really linked to each other.

Types not in Include (if include is set) are excluded implicitly.